source/examples/generated/node/data-types.codeblock.check-if-set-has-items.js

```diff
@@ -1,5 +1,5 @@
-// check if the characterTwo has completed level 3 by calling the `Realm.Set.has()` method
-const characterTwoHasCompletedLevelThree = characterTwo.levelsCompleted.has(3);
+// check if the playerTwo has completed level 3 by calling the `set.has()` method
```

Comment:
```suggestion
// check if playerTwo has completed level 3 by calling the `set.has()` method
```

Reply:
```suggestion
// check if playerTwo has completed level 3 by calling the `Set.has()` method
```

Should `Set` be capitalized?

Reply:
Similar nit throughout